src/components/Checkbox/Checkbox.tsx

```diff
@@ -72,40 +77,51 @@ export class Checkbox extends React.Component<CheckboxProps> {
 
   render() {
     const {
+      theme,
       checked,
       disabled,
       label,
       error,
       indeterminate,
       onChange,
       name,
+      suffix,
       className,
     } = this.props;
     const iconContent = this._renderIcon();
 
     return (
-      <CoreCheckbox
-        className={st(classes.root, { checked, disabled, error }, className)}
-        {...this.getDataAttributes()}
+      <div
+        className={st(
+          classes.root,
+          { box: theme === 'box', checked, disabled, error },
+          className,
+        )}
         data-hook={this.props['data-hook']}
-        checkedIcon={iconContent}
-        uncheckedIcon={iconContent}
-        indeterminateIcon={iconContent}
-        indeterminate={indeterminate}
-        checked={checked}
-        onChange={onChange}
-        name={name}
+        {...this.getDataAttributes()}
       >
-        <>
-          {!!label && <span className={classes.divider} />}
+        <CoreCheckbox
+          className={classes.core}
+          checkedIcon={iconContent}
+          uncheckedIcon={iconContent}
+          indeterminateIcon={iconContent}
+          indeterminate={indeterminate}
+          checked={checked}
+          onChange={onChange}
+          name={name}
+          disabled={disabled}
+        >
           <div
             data-hook={CHECKBOX_DATA_HOOKS.LabelWrapper}
-            className={classes.label}
+            className={`${classes.label} ${suffix && classes.suffixed}`}
```

Comment:
We use `classnames` in this library, so you can write this as:
```suggestion
            className={classnames(classes.label, {[classes.suffixed]:  suffix })}
```
and of course you'll need to add an import